examples/load-tester/load/utils/utils.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
```

Comment:
From [Style guideline for Go packages](https://rakyll.org/style-packages/):

> Short, but representative names
Package names should be short, but should be unique and representative. Users of the package should be able to grasp its purpose from just the package’s name.
Avoid overly broad package names like “common” and “util”.

I think something like `opensaves` could be appropriate here since we're interfacing with a package that wraps basic opensaves functionality.